src/Juvix/Compiler/Concrete/Data/Highlight/Input.hs

```diff
@@ -23,4 +23,4 @@ filterInput absPth HighlightInput {..} =
     }
 
 filterByLoc :: HasLoc p => Path Abs File -> [p] -> [p]
-filterByLoc p = filter ((== toFilePath p) . (^. intervalFile) . getLoc)
+filterByLoc p = filter ((== p) . (^. intervalFile) . getLoc)
```

Comment:
Why do we have this function here? it doesn't seem the right place to be. It was here before I can see. Maybe we should think to put this in Juvix.Data.Loc, or in Juvix.Extra, somewhere.

Reply:
I've moved it to `Juvix.Data.Loc`